tensorflow/core/kernels/constant_op.cc

```diff
@@ -358,6 +359,26 @@ void PlaceholderOp::Compute(OpKernelContext* ctx) {
 REGISTER_KERNEL_BUILDER(Name("Placeholder").Device(DEVICE_CPU), PlaceholderOp);
 REGISTER_KERNEL_BUILDER(Name("PlaceholderV2").Device(DEVICE_CPU),
                         PlaceholderOp);
+
+REGISTER_KERNEL_BUILDER(Name("Fill")
+                            .Device(DEVICE_DEFAULT)
+                            .TypeConstraint<int32>("T")
+                            .TypeConstraint<int32>("index_type")
+                            .HostMemory("dims")
+                            .HostMemory("value")
+                            .HostMemory("output"),
+                        FillOp<CPUDevice, int32, int32>);
+REGISTER_KERNEL_BUILDER(Name("OnesLike")
+                            .Device(DEVICE_DEFAULT)
+                            .TypeConstraint<int32>("T")
+                            .HostMemory("y"),
+                        OnesLikeOp<CPUDevice, int32>);
+REGISTER_KERNEL_BUILDER(Name("ZerosLike")
+                            .Device(DEVICE_DEFAULT)
+                            .TypeConstraint<int32>("T")
+                            .HostMemory("y"),
+                        ZerosLikeOp<CPUDevice, int32>);
```

Comment:
Nit: Could you please group them with the registrations for other devices, e.g., put "Fill" with other "Fill", "OnesLike" with "OnesLike", and "ZerosLike" with "ZerosLike"? It's easier when we skim for registrations for a specific kernel this way. Thank you!

Reply:
Thanks for the review @penpornk !. Done.